src/chrome-launcher.ts

```diff
@@ -50,7 +52,7 @@ export interface LaunchedChrome {
 
 export interface ModuleOverrides {
   fs?: typeof fs;
-  rimraf?: typeof rimraf;
+  rimraf?: RimrafModule;
```

Comment:
why'd we need this?

Reply:
well if we want our users to not need `@types/rimraf` then we can't do `import 'rimraf'` in our `d.ts` files.

because we export this type here, we need to replace this. it's just one function so it seems pretty easy to type ourselves

Reply:
oh, lol, just noticed the interface. Oh `chrome-launcher`...